genie-web/build.gradle

```diff
@@ -55,6 +55,9 @@ dependencies {
     compile("org.springframework.cloud:spring-cloud-starter-zookeeper")
     compile("org.springframework.integration:spring-integration-zookeeper")
     compile("org.springframework.retry:spring-retry")
+    compile("org.apache.maven:maven-artifact:3.5.4") {
```

Comment:
Generally we don't include versions in the subproject `build.gradle` and instead define them in the dependency management section of the parent build file so that we can ensure the same version is used across the project if it needs to be